### PR TITLE
[4.0] Append button ML-2 (RTL)

### PR DESCRIPTION
**Pull request by brianteeman**
We had an override for MR-2 (margin right) but not for ML-2 (margin left)

This can be seen on the admin login screen

### Before LTR
image

### Before RTL
image

### After LTR
image

### After RTL
image

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 3c2017e33113d3849573a0bd12c12516c6f2fb51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27409">issues.joomla.org/tracker/joomla-cms/27409</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 3c2017e33113d3849573a0bd12c12516c6f2fb51<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27409">issues.joomla.org/tracker/joomla-cms/27409</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27409">issues.joomla.org/tracker/joomla-cms/27409</a>.</sub>

File: administrator/templates/atum/scss/template-rtl.scss [hunk 1]
```diff
@@ -181,6 +181,10 @@ body {
   margin-left: auto !important;
 }
 
+.ml-2 {
+  margin-right: .5rem;
+  margin-left: 0 !important;
+}
 .mr-2 {
```
Comment (infograf768) [inline administrator/templates/atum/scss/template-rtl.scss:189]: Please add empty line before .mr-2

Comment (Quy) [reply administrator/templates/atum/scss/template-rtl.scss:189]: Line 188 is empty line.

Comment (infograf768) [reply administrator/templates/atum/scss/template-rtl.scss:189]: oops.

**Comment by infograf768** [comment]
@brianteeman @Quy 
This has created an issue as `ml-2` is not only used for the login.
`ml-2` is used instead of `mr-2` for RTL in `chromes/well.php`
`$margin = Factory::getLanguage()->isRtl() ? ' ml-2' : ' mr-2';`

Therefore
`margin-left: 0 !important;` is wrong as we get
Screen Shot 2020-01-07 at 07 46 48

instead of 
Screen Shot 2020-01-07 at 07 48 20

Maybe we should change $margin in `well.php` to
`$margin = ' mr-2';`

or delete that line and use directly
`$headerIcon = '<span class="' . htmlspecialchars($params->get('header_icon')) . ' mr-2" aria-hidden="true"></span>';`

What do you think?

**Comment by infograf768** [comment]
### NOTE: 
Also I remark that a lot of classes are NOT present in installation template-rtl.scss to override those in template.scss, including a `ml-2`...

This has never been tested. It is a release blocker. Will create an issue.
A possible solution could be to import `atum/scss/template-rtl.scss`, but not sure.

**Comment by brianteeman** [comment]
ML-2 and MR-2 are utility classes and their values in the RTL css are now correct. If there are now other issues being observed as a result of correcting those utility classes the problem is in the other classes not in these

**Comment by infograf768** [comment]
As I demonstrated, https://github.com/joomla/joomla-cms/pull/27409#issuecomment-571464842, classes are not concerned in this case.
We have to modify the chrome.
Will make a PR.

**Pull request merged**